### ISSUE TITLE
Regarding output of the file generated

### ISSUE THREAD

**Issue by Proy321** [liguowang/CrossMap#77]
Hello @liguowang
Even after replacing the first column of 'chr1' with NC_009919.1, it is still throwing the same error(attached herewith)
[New bed file.txt](https://github.com/user-attachments/files/17032571/New.bed.file.txt)

"Start coordinate is not an integer. skip chrom start end name
score strand thickStart thickEnd itemRgb blockCount
blockSizes blockStarts"

It would be nice if you can help me with the same.

**Comment by liguowang** [comment]
This is jut a warning, you still get the output file. From your attached file, I see three were converted succesfully, two were "failed".

**Comment by Proy321** [comment]
Yes @liguowang , there are multiple instances of failed rows as well, what could be the potential reason for the same,can this output be still considered.
Thanks
